lib/trip.rb

```diff
@@ -28,13 +30,39 @@ def initialize(
         raise ArgumentError, 'Passenger or passenger_id is required'
       end
 
+      unless passenger_id == nil
+        if passenger_id < 1
+          raise ArgumentError
+        end
+      end
```

Comment:
Can be simplified as:

```suggestion
      raise ArgumentError, "Bad passenger_id: #{passenger_id}" unless passenger_id.nil? || passenger_id >= 1
```